contrib/zsh-completion/_terraforming

```diff
@@ -0,0 +1,93 @@
+#compdef terraforming
+
+function _terraforming() {
+  local context curcontext=$curcontext state line
+  typeset -A opt_args
+  local ret=1
+
+  _arguments -C \
+    '1: :__terraforming_sub_commands' \
+    '*:: :->args' \
+    && ret=0
+
+  case $state in
+    (args)
+      case $words[1] in
+        (asg|dbpg|dbsg|dbsn|ec2|ecc|ecsn|eip|elb|iamg|iamgm|iamgp|iamip|iamp|iamr|iamrp|iamu|iamup|igw|nacl|nif|r53r|r53z|rds|rs|rt|rta|s3|sg|sn|vpc)
+          _arguments -C \
+            '--merge=[(TFSTATE) tfstate file to merge]:tfstate file to merge:_files' \
+            '--overwrite[Overwrite existing tfstate]' \
+            '--no-overwrite[Do not overwrite existing tfstate]' \
+            '--tfstate[Generate tfstate]' \
+            '--no-tfstate[Do not generate tfstate]' \
+            '--profile[(PROFILE) AWS credentials profile]:AWS credentials profile:__profiles' \
+            && ret=0
+        ;;
+        (help)
+          _arguments -C \
+            '1: :__terraforming_sub_commands' \
+            '(-)*:: :->null_state' \
+            && ret=0
+        ;;
+      esac
```

Comment:
It seems that only `help` command requires the special completion.
How is this? :

```bash
case $state in
  help)
    # ...
  ;;
  *)
    # ...
  ;;
esac